.conda/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
     - pillow >=9.2.0
     - h5py >=3.1.0, <4.0.0
     - opencv >=4.5.0, <5.0.0
-    - pypdfium2-team::pypdfium2_helpers >=4.0.0, <5.0.0
+    - ypdfium2-team::pypdfium2_helpers >=4.0.0, <5.0.0
```

Comment:
@justinjosephmkj 
pypdfium2 and 4.11.0 😅

Reply:
```
pypdfium2-team::pypdfium2_helpers >=4.11.0, <5.0.0
```

Reply:
@felixdittrich92 Done.